src/dev-guide/database-architecture.md

```diff
@@ -0,0 +1,28 @@
+# Database
+
+
+The architecture for the database can be seen in the following
+UML diagram:
+
+![](file:///C:/Users/Judy/AppData/Local/Temp/msohtmlclip1/01/clip_image002.png)![1666253938929](image/database-architecture/1666253938929.png)
+
+The Plot_Collection, File_Pointer and Base_Parameter tables
```

Comment:
Use code style when referring to folders and files in the codebase.e.g.

> Plot – Focuses on the business logic for plotting and visualising data.

becomes

> `Plot` – Focuses on the business logic for plotting and visualising data.